### DISCUSSION TITLE
Error: Request failed with status code 404

### DISCUSSION THREAD

**Discussion by yij77** [FlowiseAI/Flowise#756] [Q&A]
Hi, I run into an error when botpress try to access to the flowise API. Plz help me if there is anyhing wrong with my code or API. Thank you so much!
GET http://localhost:3000/api/v1/prediction/2fc77fd8-ac05-403f-a653-867e0c99bcd1: 404
Response:
:"Not found because of proxy error: Error: connect ECONNREFUSED 127.0.0.1:3000"
Error: Request failed with status code 404

Execute code:
const question = workflow.question;
const apiUrl = http://localhost:3000/api/v1/prediction/2fc77fd8-ac05-403f-a653-867e0c99bcd1;
const response = await axios.get(apiUrl);
workflow.apiResponse = _.get(response, 'data.answer', '');

**Comment by chungyau97** [comment]
Hi @yij77,

I can see that you're trying to access a **localhost:3000**.
You're trying to access from the same device or different device?

**Reply by yij77** [reply]
Hi chungyau, I am trying on the same device. I dont know why it doesnt work.

**Reply by chungyau97** [reply]
Hi @yij77,

Could you download and try this simple postman request. 

1. Change the file extension `.txt` to `.json`, then import into your postman
2. Modify the input `{{hosted_url}}`, `{{flow_id}}` and `{{api_key}}`

[Share.postman_collection.txt](https://github.com/FlowiseAI/Flowise/files/12329424/Share.postman_collection.txt)

**Reply by HenryHengZJ** [reply]
you'll have to deploy Flowise on cloud services, and use the https endpoint and use in botpress. For example: https://my-flowise-instance.onrender.com/
